src/graphic/style_manager.cc

```diff
@@ -90,6 +90,10 @@ void check_completeness(const std::string& name, size_t map_size, size_t last_en
 }
 }  // namespace
 
+StyleManager::StyleManager() {
+	init();
+}
+
 void StyleManager::init() {
```

Comment:
This function is still public. Please copy/paste its contents to the constructor and get rid of it completely.

Reply:
Done.